documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -67,11 +79,31 @@ data:
   user.password: # Password for protecting the PKCS #12 archive file
 ----
 
-== SCRAM-SHA-512 Authentication
+== TLS client authentication using an external certificate
```

Comment:
I think _external certificate_ might be confusing. It still needs to be a certificate signed by the Clients CA. It is just expected to be signed by someone / something else than the User Operator. I guess it is hard to put it into title. But I'm afraid the current title would confuse people into thinking that they can use this to bring their own custom certificate with this which is not through. It is just _issued outside of User Operator_.

Reply:
We can change to that then: _TLS client authentication using a certificate issued outside the User Operator_